x-pack/plugins/security_solution/public/resolver/view/index.test.tsx

```diff
@@ -0,0 +1,253 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { render, RenderResult } from '@testing-library/react';
+import { DataAccessLayerContext } from '../data_access_layer/context';
+import { ResolverWithoutDataAccessLayer } from '.';
+import { mockDataAccessLayer } from '../data_access_layer/mock';
+import { KibanaContextProvider } from '../../../../../../src/plugins/kibana_react/public';
+import { coreMock } from '../../../../../../src/core/public/mocks';
+import { CoreStart } from '../../../../../../src/core/public';
+import { Router } from 'react-router-dom';
+import { createMemoryHistory } from 'history';
+import { I18nProvider } from '@kbn/i18n/react';
+
+describe('resolver with a fake data layer', () => {
+  let reactRenderResult: RenderResult;
+
+  beforeEach(() => {
+    const dataAccessLayer = mockDataAccessLayer();
+    const coreStart: CoreStart = coreMock.createStart();
+
+    const history = createMemoryHistory();
+    reactRenderResult = render(
```

Comment:
Wonder if we should put all our resolver tests in this file, or break them apart. I can see this file becoming massive as functionality of resolver expands... Thoughts?

Reply:
this is the only test.

Reply:
no more tests allowed